### PR TITLE
[UBS] - rename from location_status to tariff_status in tariffs_info table #4874

### LINKED ISSUE #4874: [UBS DB] - change name of column
Should change name location_status to tariff_status in table tariffs_info.

### PR DESCRIPTION
**Pull request by sergoliarnik**
# UBS rename from location_status to tariff_status in tariffs_info table 

## Summary Of Issue :
Should change name location_status to tariff_status in table tariffs_info

## Issue Link :
https://github.com/ita-social-projects/GreenCity/issues/4874

## Summary Of Changes :
1. rename from location_status to tariff_status in tariffs_info table
2. rename from locationStatus to tariffStatus in code
3. create TariffStatus enum with the same fields as in the LocationStatus

## Code reviewers

- [ ] @github_username

## Added
- TariffStatus enum

## Changed 
- tariffs_info table
- rename from location_status to tariff_status in tariffs_info table
- rename from locationStatus to tariffStatus in code

## CHECK LIST
- [x] Code is up-to-date with the `dev` branch.
- [x] You've successfully built and run the tests locally.
- [x] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (ISSUE-https://github.com/ita-social-projects/GreenCity/issues/4874: Ticket title)
- [x] This template filled (above this section).
- [ ] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells ar duplication
- [x] `NEED_REVIEW` and `READY_FOR_REVIEW` labels are added.
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

File: dao/src/main/java/greencity/enums/TariffStatus.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public enum TariffStatus {
+    ACTIVE(1, "Активно"), NEW(2, "Новостворено"), DEACTIVATED(3, "Неактивно");
```
Comment (RomaMocherniuk) [inline dao/src/main/java/greencity/enums/TariffStatus.java]: Why do you use Ukrainian here?

File: dao/src/main/java/greencity/enums/TariffStatus.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    private int priority;
+    private String status;
```
Comment (RomaMocherniuk) [inline dao/src/main/java/greencity/enums/TariffStatus.java]: `priority` and `status` are never used
Why do we need these fields?

File: dao/src/main/java/greencity/enums/TariffStatus.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package greencity.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
```
Comment (RomaMocherniuk) [inline dao/src/main/java/greencity/enums/TariffStatus.java]: Constructor of enum should be `private`

**Review by RomaMocherniuk — APPROVED** [review]

**Review by RomaMocherniuk — CHANGES_REQUESTED** [review]
Please add tests for `TariffsInfoSpecification.toPredicate()` mrthod

**Review by JJJazl — APPROVED** [review]

**Review by juseti — APPROVED** [review]

**Review by MaksymGolik — APPROVED** [review]

**Pull request merged**